test/expr/unary/selector/selector.swift

```diff
@@ -116,7 +116,7 @@ func testParseErrors3(_ c1: C1) {
 
 func testParseErrors4() {
   // Subscripts
-  _ = #selector(C1.subscript) // expected-error{{type 'C1.Type' has no subscript members}}
+  _ = #selector(C1.subscript) // expected-error{{type 'C1' has no member 'subscript'}}
```

Comment:
I think my point is that, from my point of view, whether or not we want to support selectors for subscripts is nothing to worry about for now and that we will only need to worry about how to implement it when the time comes.
Or maybe let's ask the other way round: What would you want to be changed here?